rust/operator-binary/src/config/jvm.rs

```diff
@@ -27,8 +27,8 @@ pub enum Error {
         unit: String,
     },
 
-    #[snafu(display("the Trino version {version} is not supported, as we don't know the needed JVm configuration"))]
-    TrinoVersionNotSupported { version: String },
+    #[snafu(display("Trino version {version} is not yet supported, as the correct JVM configuration is not yet known"))]
```

Comment:
We skip like 90% of the Trino versions. Having the word `yet` kind of implicates that we will support it at some later point, which I'm a bit uncomfortable hinting at.
However, this is a nit, feel free to ignore

Reply:
I like the first one!

Reply:
Done in 4480cb8